strabo/private_helper.py

```diff
@@ -24,14 +24,18 @@ def fill_interest_point(ip,image_ids,form_title,form_body,form_geo_obj,form_laye
     ip.icon = form_icon
     ip.images = [db.session.query(schema.Images).get(int(id)) for id in image_ids]
 
-def fill_image(image,form_file_obj,form_descrip,year,month,day):
+def fill_image(image,form_file_obj,form_descrip,year,month):
     '''
-    if a flask.files object is passed in, then  
+    If a flask.files object is passed in, then it saves the new file and stores the new information in the database.
+    If the image row already stored infromation about an old image (the image was edited), then that old image is deleted.
     '''
-    image.taken_at = datetime.date(utils.safe_pos_int_conv(year),utils.safe_pos_int_conv(month),utils.safe_pos_int_conv(day))
+    image.taken_at = datetime.date(utils.safe_pos_int_conv(year),utils.safe_pos_int_conv(month),1)
     image.description = form_descrip
 
     if form_file_obj:
```

Comment:
Sorry, I thought that descrip was far more clear than desc. To me, desc can easily be thought to be all sorts of words like "descendant". If you think that desc is more clear than descrip then the argument above doesn't work, and desc should always be used unless you want to type out the whole 11 characters of "description".